server/cli.js

```diff
@@ -1,115 +1,120 @@
-    type: "string",
-  })
-  .option("end-date", {
-    alias: "e",
-    description: "Set the end date of the report",
-    type: "string",
-  })
-  .option("days-back", {
-    alias: "b",
-    description: "An alternative to start-date",
-    type: "number",
-  })
-  .option("report", {
-    alias: "r",
-    description: "Set the report type",
-    choices: ["commission", "attendance"],
-  })
-  .option("format", {
-    alias: "f",
-    description: "Set the report format",
-    choices: ["csv", "json"],
-    default: "csv",
-  })
-  .option("filename", {
-    alias: "n",
-    description: "Set file name of the report",
-    type: "string",
-  })
-  .conflicts("start-date", "days-back")
-  .demandOption(["report"], "No report selected")
-  .help()
-  .alias("help", "h")
-  .argv;
-
-function writeToFile(report, options) {
-  console.log("Writing " + options.filename);
-  fs.writeFile(options.filename, report, (error) => {
-    if (error) {
-      console.error(error);
-    }
-  });
-}
-
-function getOptions() {
-  var options = {format: argv.format};
-
-  if (argv["end-date"]) {
-    options.endDate = new Date(argv["end-date"]);
-    if (isNaN(options.endDate)) {
-      console.error("ERROR: invalid end date");
-      return null;
-    }
-  } else {
-    options.endDate = new Date();
-  }
-
-  if (argv["start-date"]) {
-    options.startDate = argv["start-date"];
-  } else if (argv["days-back"]) {
-    let date = new Date();
-    let daysback = argv["days-back"];
-    date.setDate(options.endDate.getDate() - daysback);
-    options.startDate = date;
-  } else {
-    let date = new Date();
-    const week = 7;
-    date.setDate(date.getDate() - week);
-    options.startDate = date;
-  }
-
-  if (options.startDate >= options.endDate) {
-    console.error("ERROR: the start date is greater than the end date");
-    return null;
-  }
-
-  if (!argv.filename) {
-    //this default filename looks like:  attendance 2020-04-22 - 2020-05-12.csv
-    let startDateString = (typeof options.startDate === "string") ? options.startDate : options.startDate.toISOString().substring(0, 10);
-    let endDateString = (typeof options.endDate === "string") ? options.endDate : options.endDate.toISOString().substring(0, 10);
-    //console.log("Trying filename: " + argv.report + " " + startDateString + " - " + endDateString + "." + options.format);
-    options.filename = argv.report + " " + startDateString + " - " + endDateString + "." + options.format;
-  }
-
-  return options;
-}
-
-var options = getOptions();
-if (options) {
-  console.log(options);
-  if (argv.report === "commission") {
-    getCommissionReport(options).then((report) => {
-      writeToFile(report, options);
-    });
-  } else if (argv.report === "attendance") {
-    getAttendanceReport(options.format, options.startDate, options.endDate).then((report) => {
-      writeToFile(report, options);
-    });
-  }
-}
+import {getCommissionReport} from "./reports/commission/commission-endpoint.js";
```

Comment:
Man, these line ending changes are so lame. They obscure the real changes in this large mass of false positives. Oh, well. Hopefully we'll figure out how to get all the line changes consistent soon. I'll propose something at tonight's meeting.